### PR TITLE
Release/1.0.2

### PR DESCRIPTION
**Pull request by chrk623**

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -68,8 +69,8 @@ shinyServer(function(input, output, session) {
   values$data.current.dname = NULL # current data/sheet names
   values$data.restore = NULL
   values$name.restore = NULL
-  values$lite.version = ""
-  values$lite.update = ""
+  values$lite.version = desc[, "Version"][[1]]
+  values$lite.update = "20 February 2023"
```
Comment (tmelliott) [inline server.R:73]: This value should remain blank - it gets set in the VARS file, and written/modified at build time (i.e., in the Dockerfile). It won't affect the built version, but I'm not a fan of hard-coding values in an app like this.

**Review by tmelliott — CHANGES_REQUESTED** [review]

File: server.R (hunk 1 above)
Comment (chrk623) [reply server.R]: Hey, check https://github.com/iNZightVIT/Lite/pull/296/commits/a96a1d4bca6c0a2e30f70598ef222a2928ee1de6.

Comment (tmelliott) [reply server.R]: It's OK to grab the version number from the DESCRIPTION file - I only meant the `lite.update` value (but GH decided to include both lines in my comment for some reason..) - sorry!

You also don't need to set the `lite.update` value in the `VARS.default` file (or anywhere) - it gets written at build time (in the Dockerfile), and if it's an empty string, the app doesn't print the date/version. You can test things by creating your own `VARS` file with test values - it gets ignored by git and won't get committed.

I hope that makes sense - let me know if you need further clarification!

Comment (tmelliott) [reply server.R]: Yes, that sounds about right. lite.version should pull from the DESCRIPTION, as you had initially.

**Review by tmelliott — APPROVED** [review]

**Comment by tmelliott** [comment]
BTW, I've set up a Railway instance to track this branch - https://lite-staging.up.railway.app/

**Comment by chrk623** [comment]
Also fixed #297.

**Comment by tmelliott** [comment]
Great - can you also add the VIT fix to NEWS.

**Comment by chrk623** [comment]
Oh forgot about it!
Should be good to go now.

**Comment by tmelliott** [comment]
Great - I'm fine with this going live today. From here,

- create a new PR (release/1.0.2 -> dev)
- merge the current PR (#296) - the new docker image should build automatically (GitHub actions) and deploy *at some time* once the AWS automation picks it up
- merge the other PR (this branch back into dev) and delete this branch
- tag the master branch (`git checkout master`) with the version (`git tag 1.0.2`) and push (`git push --tags`) - I've copied a GH action to create a GH release from this tag automatically

and that should be everything.

**Comment by chrk623** [comment]
Hey what do you mean by *create a new PR (release/1.0.2 -> dev)*? This branch is already merging to dev.

I'll push the release after #298.

**Comment by tmelliott** [comment]
Ah - sorry, missed that. For release branches, I usually create two PRs:
- release/x -> master (the main PR)
- release/x -> dev (to all finalised changes get merged back into dev)

This way, you can keep merging new changes into dev even after a release is started (once you start a release, you typically only make bug fixes and minor changes, like wording, etc.). Though, when it's just one person developing it's less of an issue..!

So, 
- create a new PR `release/1.0.2 -> master` - then merge that one first once it's checked and working, and *then* this one.

**Comment by tmelliott** [comment]
Also - I think #298 might be a bit of work, so let's finish this release (and check the automations work nicely) and then start working on that.

**Pull request merged**